assets/sass/patterns/molecules/article-section.scss

```diff
@@ -373,6 +373,62 @@ $arrow-size: 5;
   }
 }
 
+.section-wrapper {
```

Comment:
A max-width and margin: 0 auto; may be needed here, as in the .content-header.

Reply:
The class "section-wrapper" is updated with these styles, the content header will be handled in another task, now I am reusing an existing variant just for visual presentation